OData/src/System.Web.OData/OData/Query/FilterQueryOption.cs

```diff
@@ -109,6 +109,16 @@ public FilterClause FilterClause
         }
 
         /// <summary>
+        /// Returns a new instance of the vistor to use to transform the filterclause's expression into the actual 
+        /// expression to use 
+        /// </summary>
+        /// <param name="queryOptionsParser">The <see cref="ODataQueryOptionParser"/> which is used to parse the query option</param>
+        /// <returns>A new visitor instance</returns>
+        protected virtual Microsoft.OData.Core.UriParser.Visitors.QueryNodeVisitor<QueryNode> GetVisitor(ODataQueryOptionParser queryOptionsParser) {
```

Comment:
As before it allows us to get a custom visitor of our choosing instead of the fixed one which is not possible to override.